blockchain/vm/contracts.go

```diff
@@ -216,6 +217,28 @@ func ActivePrecompiles(rules params.Rules) []common.Address {
 	}
 }
 
+// IsPrecompiledContractAddress returns true if this is used for TestExecutionSpecState and the input address is one of precompiled contract addresses.
+func IsPrecompiledContractAddress(addr common.Address, rules params.Rules) bool {
+	if relaxPrecompileRangeForTest {
+		rules.IsIstanbul = false
+		activePrecompiles := ActivePrecompiles(rules)
```

Comment:
Why specifically IsIstanbul=false? What is the intended value of `activePrecompiles`? Is it PrecompiledAddressesByzantium?

Reply:
https://github.com/ulbqb/kaia/blob/acb34de13dbef4d6dbb6f0d2d8000304170c6787/blockchain/vm/contracts.go#L209-L217
`ActivePrecompiles` returns inactive precompiled contracts, 0x0a and 0x0b If IsIstanbul = true.